docs/developer-docs/multi-chain/overview.mdx

```diff
@@ -0,0 +1,115 @@
+
+[Learn more about how the Ethereum integration works](/docs/current/developer-docs/multi-chain/ethereum/overview).
+
+## Benefits of multi-chain
+
+When compared to traditional blockchain bridges, ICP's multi-chain capabilities have several benefits:
+
+- **Enhanced programmability**: Dapps can expand their functionalities and capabilities beyond just ICP, allowing for new use-cases and applications.
+
+- **Orchestration**: Transactions on integrated networks can be orchestrated and automated using ICP smart contracts.
+
+- **Security and privacy**: ICP's direct integration with other chains is more secure than using an intermediary blockchain bridge, which may have security vulnerabilities and be exploited.
+
+- **Lower transaction costs**: Submitting transactions to Bitcoin or Ethereum from an ICP smart contract have a significantly lower cost than submitting transactions on the chains directly.
+
+- **Faster transaction times**: Transaction times on Bitcoin and Ethereum are drastically reduced by using the ICP multi-chain integrations for each.
+
+## Multi-chain integrations
+
+Current multi-chain integrations include Bitcoin and Ethereum, and the EVM RPC canister can be used to interact with any EVM-compatible chain, such as Polygon or Avalanche.  Direct integration with other chains is possible, and developers can build their own.
+
+## Multi-chain tokens
+
+ICP supports multi-chain tokens that are created using chain-key cryptography. These tokens are digital twins of their corresponding native token and are backed 1:1 by their native token. that is held 100% on-chain on ICP. For example, ckBTC is the digital twin of BTC, and ckETH is the digital twin of ETH.
+
+Chain-key tokens are different from traditional wrapped tokens. Chain-key tokens use advanced cryptography to create an identical corresponding token with the same properties and valuation, but hosted on ICP. Traditional wrapped tokens use underlying assets with different properties.
+
+Chain-key tokens don't use blockchain bridges, intermediaries, or centralized entities. All chain-key operations are performed on-chain. They also have a faster finality time and a cheaper transaction cost than wrapped tokens typically have.
+
+Currently supported multi-chain tokens include:
+
+- [ckBTC](/docs/current/developer-docs/multi-chain/bitcoin/ckbtc)
+
+- [ckETH](/docs/current/developer-docs/multi-chain/ethereum/cketh)
+
+In the future, chain-key tokens are planned for:
+
+- ckERC-20
```

Comment:
This shouldn't be here. If at all, then as a third point and rather as "And other ERC20 twins"

Reply:
Revised to "In the future, chain-key tokens are planned for other ERC20 twins."